package.json

```diff
@@ -139,7 +139,8 @@
     },
     "afterSign": "./notarization/afterSignHook.js",
     "dmg": {
-      "icon": "build/icon.png"
+      "icon": "build/icon.png",
+      "writeUpdateInfo": false
```

Comment:
Do we need to add this to disable blockmaps? This may harm automatic updates if that doesn't produce update.yml file.

Reply:
The latest.yml still exists. Let's see if it works, otherwise we can remove those entries again.